.nvmrc

```
@@ -1 +1 @@
-12.22
+16.16.0
```

I think we can put just 16, and it should be better. `16.16.0` is a very recent version, and probably few folks are using it.

```suggestion
16
```

---

👍🏼 Make sense! Get updated.